packages/malloy-vscode/src/common/connection_manager.ts

```diff
@@ -133,4 +137,15 @@ export class ConnectionManager {
       }
     }
   }
+
+  getConnectionManager(url: URL): LookupConnection<Connection> {
```

Comment:
Maybe this should be called `getConnectionMap` or something, and `.connections` should be made private.